lib/frontend/post-github-configuration.js

```diff
@@ -60,6 +60,7 @@ module.exports = async (req, res) => {
       clientKey: getHashedKey(req.body.clientKey),
       installationId: req.body.installationId,
       host: req.session.jiraHost,
+      githubHost: new URL(userInstallation.html_url).hostname,
```

Comment:
you have already hostname at line no 16, see if you can re-use of make common code

Reply:
these are different files.
this is just one line of code so didnt make it common.